earth_enterprise/rpms/opengee-fusion/snippets/post-install.sh

```diff
@@ -67,6 +69,97 @@ main_postinstall()
 # Post-install Functions
 #-----------------------------------------------------------------
 
+fix_symlinks()
```

Comment:
Please, fix symlinks in the Gradle file instead of doing this.  This is not as clean, as not secure.

Look at the `link` commands in the `openGeeServerRpm` task for examples of packaging symlinks.

Reply:
A better way to fix it is to add an option to the `GeeRpm` and `GeeDeb` classes to scan for symlinks in a package, and add them as symlinks before packaging.

Listing them manually is OK for a hot fix.  However, it will be prone to getting out of date and creating security issues anytime this list of symlinks changes, and a corresponding update isn't made in the Gradle scripts.

Reply:
Right. I am going to look into it. The reason this ticket came up is due to Gradle handling the links incorrectly.

Reply:
Looks like I will be able to do it this way. Will take a little bit to implement everything in Gradle, but it looks doable.

Reply:
Great.

Also, I may be able to give some answers, if Gradle questions come up.

Reply:
Have made adjustments so symlinks are fixed in Gradle instead of the post install script.